src/_common-loader.ts

```
@@ -0,0 +1,200 @@
+	const allEntries: ProcessedEntry[] = [];
+
+	for (const { file } of sortedFilesWithBase) {
+		const content = await readFile(file, 'utf-8');
+		const lines = content
+			.trim()
+			.split('\n')
+			.filter(line => line.length > 0);
+
+		for (const line of lines) {
+			try {
+				const parsed = JSON.parse(line) as unknown;
```

ESLint

[error] Unsafe argument of type error typed assigned to a parameter of type `string`.

(ts/no-unsafe-argument)